### DISCUSSION TITLE
Arsha Theme example published for Oqtane 5.1

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4035] [General]
The full source code and instructions are available here:

https://github.com/oqtane/Oqtane.Theme.Arsha

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/b6e82b53-4c04-4549-bda6-e3cc6632fe79)

**Comment by thabaum** [comment]
Love it!

**Comment by mdmontesinos** [comment]
Thanks for the repo!! It is especially useful to understand how to handle js with blazor enhanced navigation.
